ls1intum/Artemis
# `Programming exercises`: Update template gradle version to 7.6

### Checklist
#### General
#### Changes affecting Programming Exercises

### Motivation and Context
A new Gradle version got released.

### Description
Updates the template wrapper.

### Steps for Testing
Create a new Gradle Programming exercise and check that:
1. Executing tests on Artemis is possible
2. Importing the project locally is possible.

### Review Progress

#### Code Review
#### Manual Tests
